### PR TITLE
Support Inbox :)

### PR DESCRIPTION
**Pull request by jlevesy**

### CODE REVIEW

**Comment by jlevesy** [comment]
Still need to implement alias fetching.
We should get rid of this crappy gmail.js script, as it is pretty easy to extract all data / events we need from dom, for the sake of portability.

**Comment by jlevesy** [comment]
@AlexisMontagne @phndiaye  RTM ?

**Comment by jlevesy** [comment]
@AlexisMontagne @phndiaye  PING ?

File: src/module/inbox.coffee [hunk 1]
```diff
@@ -0,0 +1,28 @@
+api = require('app/module/inbox-api')
+signr = require('app/module/signr')
+$ = require('jquery')
+
+injectSnippet = (element, snippet) ->
+  if $(element).find('.aR')
```
Comment (phndiaye) [inline src/module/inbox.coffee]: `unless` is usable there :)

**Comment by phndiaye** [comment]
LGTM :+1: 

LGTM

**Pull request merged**